traverser/todoerrs/todoerrs.go

```diff
@@ -15,9 +15,9 @@ import (
 type TodoErrCallback func(todoerr *errors.TODO) error
 
 // NewTraverser for todo errors
-func NewTraverser(f *fetcher.Fetcher, ignoredPaths []string, callback TodoErrCallback) *Traverser {
+func NewTraverser(f *fetcher.Fetcher, ignoredPaths, todos []string, callback TodoErrCallback) *Traverser {
 	return &Traverser{
-		comments.NewTraverser(ignoredPaths, commentsCallback(checker.New(f), callback)),
+		comments.NewTraverser(ignoredPaths, commentsCallback(checker.New(f, todos), callback)),
```

Comment:
I feel like `todos` is ambiguous here, especially given that it isn't an explicit type, it is a `[]string`.

Suggest renaming it to `customTodos` for clarity